core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -272,39 +277,53 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
                         monitoringScript: Option[String],
                         monitoringImage: Option[String],
                         monitoringImageScript: Option[String]
-  ): ExecutionServiceWorkflowOptions = {
+  )(implicit executionContext: ExecutionContext): Future[ExecutionServiceWorkflowOptions] = {
     val petSAEmail = petSAJson.parseJson.asJsObject.getFields("client_email").headOption match {
       case Some(JsString(value)) => value
       case Some(x) => throw new RawlsException(s"unexpected json value for client_email [$x] in service account key")
       case None    => throw new RawlsException(s"client_email missing for service account key json")
     }
 
-    ExecutionServiceWorkflowOptions(
-      submission.submissionRoot,
+    for {
+      currentSettings <- workspaceSettingRepository.getWorkspaceSettings(workspace.id)
+      separateSubmissionSetting = currentSettings.collectFirst { case setting: SeparateSubmissionFinalOutputsSetting =>
+        setting
+      }
+      (final_workflow_outputs_dir, final_workflow_outputs_dir_metadata) = separateSubmissionSetting match {
+        case Some(setting) if setting.config.enabled =>
+          (Option(s"gs://${workspace.bucketName}/submissions/final-outputs/${submission.id}"), Option("destination"))
+        case _ => (None, None)
+      }
       // Intermediate/final output separation: location 2/2 (SU-166, WX-1702)
       // Final outputs are moved to the directory specified
       // Cromwell `/outputs` endpoint and Terra data table use this location
```

Comment:
References to "move" are obsolete, and might as well just use the new ticket number
```suggestion
      // Cromwell `/outputs` endpoint and Terra data table use this location
```